src/main/java/core/basesyntax/strategy/ReportGenerateImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportGenerateImpl implements ReportGenerate {
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity").append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.getAllFruits().entrySet()) {
+            report.append(entry.getKey()).append(",").append(entry.getValue())
+                    .append(System.lineSeparator());
```

Comment:
The header 'fruit,quantity' is hardcoded. If the report format changes, you'll need to update it here. Consider defining it as a constant or allowing it to be configurable.